optaplanner-core/src/test/java/org/optaplanner/core/api/solver/SolverManagerTest.java

```diff
@@ -37,23 +44,19 @@
 import org.optaplanner.core.impl.testdata.domain.extended.TestdataUnannotatedExtendedSolution;
 import org.optaplanner.core.impl.testdata.util.PlannerTestUtils;
 
-import static org.junit.Assert.*;
-import static org.optaplanner.core.impl.testdata.util.PlannerAssert.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+import static org.junit.Assert.assertEquals;
+import static org.junit.Assert.assertTrue;
+import static org.junit.Assert.fail;
+import static org.optaplanner.core.impl.testdata.util.PlannerAssert.assertSolutionInitialized;
 
 public class SolverManagerTest {
 
-    @Test(timeout = 600_000)
+    @Test(timeout = 1_000)
```

Comment:
Some of our jenkins machines are really slow. If the machine is too slow, this test will randomly fail if the timeout is too short. Always put it on 600_000 even if it runs in less than a second on your machine, just for safety.
The timeout is just there to prevent a forever run.

Reply:
I will unify all the SolverManager tests to 60_000. 1 minute is more than enough, as the test usually finishes below 1 second. If some of the tests takes more than a minute in CI, I would argue we should be notified about it.